src/utils/convertPipes.js

```diff
@@ -3,22 +3,32 @@ const cheerio = require("cheerio");
 const getMetadata = (ctx, metadataId) =>
   ctx.questionnaireJson.metadata.find(({ id }) => id === metadataId);
 
+const formatNumber = value => `${value} | format_number`;
+const formatDate = value => `${value} | format_date`;
+const formatCurrency = (value, unit = "GBP") =>
+  `format_currency(${value}, '${unit}')`;
+
+const { unescapePiping } = require("./HTMLUtils");
+
 const FILTER_MAP = {
-  Number: "|format_number",
-  Currency: "|format_currency",
-  Date: "|format_date",
-  DateRange: "|format_date"
+  Number: formatNumber,
+  Currency: formatCurrency,
+  Date: formatDate,
+  DateRange: formatDate
 };
 
+const renderPipedAnswer = id => `answers['answer${id}']`;
```

Comment:
Why do we need this function and `renderPipedMetadata`. I think its fine leaving them in the `PIPE_TYPES` as render methods.